src/Data/Algorand/Transaction/Signed.hs

```diff
@@ -260,20 +260,24 @@ instance FromJSON LogicSignature where
 
 data BlockTransaction = BlockTransaction
   { btSig :: TransactionSignature
+  -- , btMsig :: MultiSignature
+  -- , btLsig :: LogicSig
   , btTxn :: Transaction
+  -- , btAuthAddr :: Address
```

Comment:
Well, now I should ask "Is it supposed to be uncommented later?" about these new commented out fields :wink: 

Do you have similar links for them?

Reply:
https://github.com/algorand/go-algorand/blob/88c9f49f14d04f9f209b2acbd42c5864cae9bdc6/data/transactions/signedtxn.go#L31

))